pyatlan/model/assets/asset00.py

```diff
@@ -201,7 +201,6 @@ def validate_required(self):
     type_name: str = Field(
         "Referenceable",
         description="Name of the type definition that defines this instance.\n",
-        regex=r"^([A-Z][a-z])+$",
```

Comment:
@ErnestoLoma looks like regex piece you'd added is being dropped in this PR... was the regex piece added as part of the code generation? (Feels like it was lost as part of regenerating?)

Reply:
@cmgrote I added but then removed it. I guess I forgot to regnerate. It's not needed.